enumerate.ts

```diff
@@ -1,6 +1,6 @@
-export const enumerate = (words: string[], cutoff = 4, placeholder = 'Things'): string => {
+export const enumerate = (words: string[], cutoff = 4, placeholder = 'Things', andStr = 'and', noOxford): string => {
```

Comment:
that will compile, but better to give `noOxford` a type. Otherwise it defaults to the `any` type.

Did you try `(words: string[], cutoff = 4, placeholder = 'Things', andStr = 'and', noOxford?: boolean)`?